libs/wire-api/src/Wire/API/ErrorDescription.hs

```diff
@@ -221,6 +221,24 @@ type NotConnected = ErrorDescription 403 "not-connected" "Users are not connecte
 notConnected :: NotConnected
 notConnected = mkErrorDescription
 
+type ConnectionLimitReached = ErrorDescription 403 "connection-limit" "Too many sent/accepted connections."
+
+connectionLimitReached :: ConnectionLimitReached
+connectionLimitReached = mkErrorDescription
+
+type InvalidUser = ErrorDescription 400 "invalid-user" "Invalid user."
+
+invalidUser :: InvalidUser
+invalidUser = mkErrorDescription
+
+type NoIdentity = ErrorDescription 403 "no-identity" "The user has no verified identity (email or phone number)."
+
+-- FUTUREWORK: if that additional error number in the message has any importance
+-- (it probably doesn't matter), maybe there's a way to add it to the
+-- description here.
+noIdentity :: Int -> NoIdentity
```

Comment:
If you want to add the error code to the description, you can just use the `ErrorDescription` constructor, which takes a `Text` to use as the description, instead of the static one that is part of the type. You could do something like *(I haven't type-checked this)*:
```haskell
noIdentity :: (NoIdentity ~ ErrorDescription code lbl desc) => Int -> NoIdentity
noIdentity n = ErrorDescription (Text.pack (symbolVal (Proxy @desc)) <> " (code "<> Text.pack (show n) <> ")")
```